packages/megasync-deb/megasync-deb.pacscript

```diff
@@ -0,0 +1,70 @@
+name="megasync-deb"
+gives="megasync"
+repology=("project: megasync")
+version="4.9.4"
+build_version="-3.1"
+replace=("${gives}")
+breaks=("${gives}" "${gives}-bin" "${gives}-git" "${gives}-app" "${gives}-deb")
+incompatible=("debian:sid" "debian:buster" "debian:stretch" "ubuntu:devel" "ubuntu:18.*" "ubuntu:19.*" "ubuntu:20.*" "ubuntu:21.*" "ubuntu:23.10")
+arch=("amd64")
+depends=("libc-ares2" "libc6" "libgcc-s1" "libmediainfo0v5" "libqt5core5a" "libqt5dbus5" "libqt5gui5 | libqt5gui5-gles" "libqt5network5" "libqt5svg5" "libqt5widgets5" "libqt5x11extras5" "libsqlite3-0" "libssl3" "libstdc++6" "libudev1" "libxcb1" "libzen0v5" "apt-transport-https" "gpg")
```

Comment:
These are provided in the Deb right? In that case, you don't need them here

Reply:
The dependencies are provided by .deb files, and they are the same across the Ubuntu/Debian versions I chose